package.json

```diff
@@ -43,7 +43,7 @@
     "repository": "git://github.com/GPII/universal.git",
     "main": "./index.js",
     "engines": {
-        "node": ">=4.2.1"
+        "node": ">=6.3.1"
```

Comment:
See comment on the Linux pull request at this same spot. Does this change actually force us to use only >= Node 6.3.1, or is it also compatible with Node 4.2.x?